package.json

```diff
@@ -68,7 +68,7 @@
     "test": "npm run lint && mocha --recursive test/functional test/unit test/core",
     "test-nolint": "mocha --recursive test/functional test/unit test/core",
     "coverage": "istanbul cover mongodb-test-runner -- -t 60000 test/core test/unit test/functional",
-    "lint": "eslint lib test",
+    "lint": "eslint -v && eslint lib test",
```

Comment:
Unrelated to this PR. I just sometimes get weird discrepancies between running `eslint` locally and what runs on evergreen. Was thinking adding this could help make things a bit more explicit around the version of `eslint` being used, when it is being used.